tests/test_process_parfile.py

```diff
@@ -95,6 +95,18 @@ def test_parse_parfile_index_param():
     assert pint_dict["DMX_0025"] == ["2 1 3"]
 
 
+def test_parse_dm_derivs():
```

Comment:
Perhaps it would be good to include a test that verifies the PINT name of the parameters? Like just reference m.DM1, m.DM2, m.DM3 (and check that m.DM001 does not exist).

Reply:
I hope my change just now addresses this